### DISCUSSION TITLE
What is the significance of parameters that end in 10 or 20?

### DISCUSSION THREAD

**Discussion by bwims** [HASwitchPlate/openHASP#627] [Documentation]
Hello,

There are a number of parameters that appear to be documented that have counterparts that end in 10 or 20 for which the meaning is not documented.

As a newbie, I'd like to know the significance as I am working through the examples.

E.g. under climate/thermometer, there are these:
line_width10
line_color10
pad_left20
pad_right20
start_angle10
end_angle10

Are they some sort of multiplier?

Thanks!

B.

**Comment by fvanroie** [comment]
The suffixes and their meaning are documented in the [Styling Properties](https://www.openhasp.com/0.7.0/design/styling/#suffixes).

**Reply by bwims** [reply]
Many thanks!

**Reply by bwims** [reply]
Just as an example, with "arc", "pad_top20" would mean "Set the padding on the top" with "the knob which can be used the change the value"

Correct?
